### PR TITLE
34u.MultiBaseCSHome version 5.1.1.182

### PR DESCRIPTION
**Pull request by Kohbach**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154485)

### CODE REVIEW

**Comment by Kohbach** [comment]
@microsoft-github-policy-service agree company="34u GmbH"

File: manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: 5.1.1.182
+InstallerType: zip
+NestedInstallerType: msi
+NestedInstallerFiles:
+  - RelativeFilePath: MultiBaseCS5Home.msi
+UpgradeBehavior: install
+Installers:
+  - Architecture: x86
+    InstallerUrl: https://www.multibasecs.de/downloads/Files/MultiBaseCS/Version_51/MSI/MultiBaseCS5Home_511182.zip
+    InstallerSha256: d37eb31eeeecb5a221923f82fe5b9aab32b573c7d830e8cf3fea3a6d5feb5d61
```
Comment (Trenly) [inline manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.installer.yaml:14]:
```suggestion
    InstallerSha256: d37eb31eeeecb5a221923f82fe5b9aab32b573c7d830e8cf3fea3a6d5feb5d61
    InstallerLocale: de-DE
    ProductCode: '{4ECEB5BA-9374-4D5C-9F77-95FBBA24DE0F}'
```

File: manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: 34u.MultiBaseCSHome
+PackageVersion: 5.1.1.182
```
Comment (Trenly) [inline manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.installer.yaml:5]:
```suggestion
PackageVersion: 4.9.0.99
```
This should match what the package writes to Control Panel. After changing this, the version folder will need to be renamed
```
--> Comparing ARP Entries

DisplayName        DisplayVersion Publisher ProductCode                            Scope
-----------        -------------- --------- -----------                            -----
MultiBaseCS 5 Home 4.9.0.99       34u GmbH  {4ECEB5BA-9374-4D5C-9F77-95FBBA24DE0F} Machine
```

File: manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.locale.de-DE.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: 34u.MultiBaseCSHome
+PackageVersion: 5.1.1.182
```
Comment (Trenly) [inline manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.locale.de-DE.yaml:5]:
```suggestion
PackageVersion: 4.9.0.99
```

File: manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: 34u.MultiBaseCSHome
+PackageVersion: 5.1.1.182
```
Comment (Trenly) [inline manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.yaml:5]:
```suggestion
PackageVersion: 4.9.0.99
```

File: manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.locale.de-DE.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: 34u.MultiBaseCSHome
+PackageVersion: 5.1.1.182
+PackageLocale: de-DE
+Publisher: 34u GmbH
+PublisherUrl: https://www.multibasecs.de
+PublisherSupportUrl: https://www.multibasecs.de/hilfe/
+PrivacyUrl: https://www.multibasecs.de/impressum/
+PackageName: MultiBaseCS Home
```
Comment (Trenly) [inline manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.locale.de-DE.yaml:11]:
```suggestion
PackageName: MultiBaseCS 5 Home
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Kohbach — COMMENTED** [review]
Change Version to 5.1.1.183

**Review by Kohbach — COMMENTED** [review]
Change to Version 5.1.1.185

**Pull request closed without merging**